go.mod

```diff
@@ -266,3 +266,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.3-0.20230601165947-6ce0bf390ce3 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace github.com/openshift/api => github.com/joelspeed/api v0.0.0-20240424135613-d3739e09825b
```

Comment:
Can you make this openshift api master rather than using a replace please

Reply:
But we've not merged the MAPI -> CAPI Migration changes? Or do they not matter for this?

Reply:
They don't matter for now, the idea is just to bring the repo up to master as it is now